src/quacc/recipes/newtonnet/core.py

```diff
@@ -0,0 +1,399 @@
+"""
+Core recipes for the NewtonNet code
+"""
+from __future__ import annotations
+
+from copy import deepcopy
+from typing import Literal
+
+import covalent as ct
+import numpy as np
+from ase.atoms import Atoms
+from ase.optimize.optimize import Optimizer
+from ase.units import _c, fs
+from ase.vibrations.data import VibrationsData
+from monty.dev import requires
+
+try:
+    from sella import IRC, Sella
+except:
+    Sella = None
+from quacc import SETTINGS
+from quacc.schemas.ase import (
+    summarize_opt_run,
+    summarize_run,
+    summarize_thermo_run,
+    summarize_vib_run,
+)
+from quacc.util.calc import run_ase_opt, run_calc
+from quacc.util.thermo import ideal_gas
+
+try:
+    from newtonnet.utils.ase_interface import MLAseCalculator as NewtonNet
+except ImportError:
+    NewtonNet = None
+
+
+def get_hessian(atoms):
+    mlcalculator = NewtonNet(
+        model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
+        settings_path=SETTINGS.NEWTONNET_CONFIG_PATH.split(":"),
+    )
+    mlcalculator.calculate(atoms)
+    return mlcalculator.results["hessian"].reshape((-1, 3 * len(atoms)))
+
+
+def add_stdev_and_hess(summary):
+    for i in range(len(summary["trajectory"])):
+        mlcalculator = NewtonNet(
+            model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
+            settings_path=SETTINGS.NEWTONNET_CONFIG_PATH.split(":"),
+        )
+        mlcalculator.calculate(summary["trajectory"][i]["atoms"])
+        summary["trajectory_results"][i]["hessian"] = mlcalculator.results["hessian"]
+        summary["trajectory_results"][i]["energy_std"] = mlcalculator.results[
+            "energy_disagreement"
+        ]
+        summary["trajectory_results"][i]["forces_std"] = mlcalculator.results[
+            "forces_disagreement"
+        ]
+        summary["trajectory_results"][i]["hessian_std"] = mlcalculator.results[
+            "hessian_disagreement"
+        ]
+    return summary
+
+
+@ct.electron
+@requires(NewtonNet, "NewtonNet must be installed. Try pip install quacc[newtonnet]")
+def static_job(
+    atoms: Atoms, newtonnet_kwargs: dict | None = None, opt_swaps: dict | None = None
+) -> dict:
+    """
+    Carry out a single-point calculation.
+
+    Parameters
+    ----------
+    atoms : Atoms
+        The atomic configuration to be relaxed.
+    newtonnet_kwargs : dict, optional
+        Additional keyword arguments for the tblite calculator. Defaults to None.
+
+    Returns
+    -------
+    dict
+        A summary of the run, including relevant information about the calculation results.
+    """
+    newtonnet_kwargs = newtonnet_kwargs or {}
+    opt_swaps = opt_swaps or {}
+    input_atoms = deepcopy(atoms)
+    # Define calculator
+    mlcalculator = NewtonNet(
+        model_path=SETTINGS.NEWTONNET_MODEL_PATH.split(":"),
```

Comment:
Remove the splitting based on ":". You should be able to simply specify a list of paths in the `settings.py` file, and it will be imported automatically as a `list[str]`.

Reply:
How will the input looks like when the ":" is removed in ".quacc.yaml"?
Will it contain the square brackets?

Reply:
I believe so. ["path1", "path2"]

Reply:
Done.